hack/run-devenv.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
maybe to be similar to the other scripts we have we can use

```
#!/usr/bin/env bash
```

just a comment non blocking

Reply:
Note it gets called from within a container, so bash won't work. I can't remember if there was a problem `/usr/bin/env sh` or not.

Reply:
ahh so ignore my comment 🤦‍♂️

Reply:
`/bin/sh` is required to exist due to POSIX anyway :p